src/main/java/eu/greev/dcbot/ticketsystem/interactions/commands/GetTickets.java

```diff
@@ -21,14 +22,15 @@
 
 public class GetTickets extends AbstractCommand {
     public static final List<ScrollEntity> PAGE_SCROLL_CACHE = new ArrayList<>();
+    public static final int PAGE_SIZE = 25; // this value shouldn't be greater than 25 because of Discord limitations
 
     public GetTickets(Config config, TicketService ticketService, EmbedBuilder missingPerm, JDA jda) {
         super(config, ticketService, missingPerm, jda);
 
         new Timer().schedule(new TimerTask() {
             @Override
             public void run() {
-                PAGE_SCROLL_CACHE.removeIf(entity -> System.currentTimeMillis() - entity.getTimeCreated() > TimeUnit.HOURS.toMillis(12));
+                PAGE_SCROLL_CACHE.removeIf(entity -> Instant.now().toEpochMilli() - entity.getTimeCreated() > TimeUnit.HOURS.toMillis(12));
```

Comment:
We can use something like this to make it more readable: `Instant.now().minus(12, Hours).isBefore(Instant.ofEpochSecond/Milli(entity.getTimeCreated()))`